test/new-e2e/tests/process/testing.go

```diff
@@ -73,6 +76,14 @@ func assertProcessCollected(
 	assert.True(t, populated, "no %s process had all data populated", process)
 }
 
+// assertProcessNotCollected asserts that the given process is NOT collected by the process check
+func assertProcessNotCollected(t *testing.T, payloads []*aggregator.ProcessPayload, process string) {
+	for _, payload := range payloads {
+		found, _ := findProcess(process, payload.Processes, false)
+		require.False(t, found, "%s process found", process)
```

Comment:
nit: the function name says it asserts, but the code calls `require`, should we keep them consistent?

Reply:
Let me update that.

Reply:
Fixed.